netneurotools/plotting.py

```diff
@@ -411,9 +414,18 @@ def plot_fsaverage(data, lhannot, rhannot, *, surf='pial', views='lat',
         # get appropriate data, accounting for hemispheric asymmetry
         currdrop = np.intersect1d(drop, names)
         if hemi == 'lh':
-            ldata, rdata = np.split(data, [len(names) - len(currdrop)])
-            if mask is not None:
-                lmask, rmask = np.split(mask, [len(names) - len(currdrop)])
+            if order == 'LR':
+                splitID = len(names) - len(currdrop)
+                ldata, rdata = np.split(data, [splitID])
+                if mask is not None:
+                    lmask, rmask = np.split(mask, [splitID])
+            elif order == 'RL':
+                splitID = len(data) - len(names) + len(currdrop)
+                rdata, ldata = np.split(data, [splitID])
+                if mask is not None:
+                    rmask, lmask = np.split(mask, [splitID])
+            else:
+                raise ValueError('Order must be either \'LR\' or \'RL\'')
```

Comment:
Can we move this higher in the function? Ideally these sorts of parameter checks should be done as early as possible so we minimize computation if the user simply made a little typo. This could go below the  `if mask is not None` line and would be rewritten as:

```python
if order not in ['LR', 'RL']:
    raise ValueError('Order must be either \'LR\' or \'RL\'')
```